File: as-1.py
#1.Write a program to input your marks for three subjects then find sum and percentage.
sub1 = int(input("Enter marks for sub1: "))
sub2 = int(input("Enter marks for sub2: "))
sub3 = int(input("Enter marks for sub3: "))
total_marks = sub1+sub2+sub3
percentage = (total_marks / 300) * 100
print("Total marks :",total_marks)
print("percentage :",percentage)

File: as-1.r
#1.Write a program to input your marks for three subjects then find sum and percentage.
math = int(input("Enter marks for math: "))
science = int(input("Enter marks for science: "))
social = int(input("Enter marks for social: "))
english = int(input("Enter marks for english: "))
hindi = int(input("Enter marks for hindi: "))
odia = int(input("Enter marks for odia: "))
total_marks = math + science + social + english + hindi + odia
percentage = (total_marks / 600) * 100
print("Total marks :",total_marks)
print("percentage :",percentage)